### PR TITLE
start.py has to kill the Java process in case of a KeyboardInterrupt (#694)

### PR DESCRIPTION
**Pull request by whimboo**
This PR fixes issue #694. @parkouss can you have a look at it? It's something we missed in your PR.

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -44,5 +44,11 @@ def start_jenkins():
 
 
 if __name__ == "__main__":
-    proc = start_jenkins()
-    sys.exit(proc.wait())
+    proc = None
+
+    try:
+        proc = start_jenkins()
```
Comment (parkouss) [inline start.py]: Maybe you can put this before the try ? You wouldn't have to handle the case when proc is None.

Comment (whimboo) [inline start.py]: Keep in mind that the download of the war file can also take some seconds. If you hit ctrl-c during that time we won't be inside the try block.

Comment (parkouss) [inline start.py]: but then, the proc won't be returned, so it will be None. So you won't kill anything, no ?

Comment (whimboo) [inline start.py]: Right. We discussed that on IRC and I implemented this change. New commit is up.

**Comment by parkouss** [comment]
r+

**Pull request closed without merging**